strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/maven/MavenArtifactController.java

```diff
@@ -341,23 +305,21 @@ private void generateDirectoryListing(Repository repository,
             response.getWriter()
                     .close();
 
-        }
-        catch (Exception e)
-        {
+        } catch (Exception e) {
             logger.error(" error accessing requested directory: " + file.getAbsolutePath(), e);
 
             response.setStatus(404);
         }
     }
 
     @ApiOperation(value = "Copies a path from one repository to another.", position = 4)
-    @ApiResponses(value = { @ApiResponse(code = 200, message = "The path was copied successfully."),
-                            @ApiResponse(code = 400, message = "Bad request."),
-                            @ApiResponse(code = 404, message = "The source/destination storageId/repositoryId/path does not exist!") })
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "The path was copied successfully."),
+            @ApiResponse(code = 400, message = "Bad request."),
+            @ApiResponse(code = 404, message = "The source/destination storageId/repositoryId/path does not exist!")})
     @PreAuthorize("hasAuthority('ARTIFACTS_COPY')")
-    @RequestMapping(produces = MediaType.TEXT_PLAIN_VALUE,
-                    value = "/copy/{path:.+}",
-                    method = RequestMethod.POST)
+    @RequestMapping(produces = MediaType.ALL_VALUE,
```

Comment:
Why have you changed this to `MediaType.ALL_VALUE`?

Reply:
I don't understand the necessity of changes around `TrashController.java` and `MavenArtifactController.java` in this PR. What's the issue with old implementation ?

Reply:
Hi!
Some tests were unsuccessful after updating dependencies.
I get the following error: java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <406>.

I tried to quickly understand the causes of the problem, but so far I have not found a solution.
For this reason, I changed MediaType for the two methods.